src/utils/utcTimestamp.ts

```diff
@@ -0,0 +1,3 @@
+// returns current UTC unix timestamp (epoch) in seconds
```

Comment:
If you use JSDoc then you get this comment in your editor intellisense:

```suggestion
/**
 * Returns current UTC unix timestamp (epoch) in seconds
 */
```

<img width="352" alt="image" src="https://user-images.githubusercontent.com/1938/64560906-53257280-d2fe-11e9-834d-0852bc60aa80.png">

Reply:
Awesome, will update.